exporter/wavefront/trace.go

```diff
@@ -0,0 +1,178 @@
+package wavefront
+
+import (
+	"bytes"
+	"encoding/hex"
+	"strconv"
+	"sync/atomic"
+
+	wfsender "github.com/wavefronthq/wavefront-sdk-go/senders"
+	"go.opencensus.io/trace"
+)
+
+const (
+	// Tags
+	spanKindKey  = "SpanKind"
```

Comment:
`span.kind` according to opentracing spec: https://github.com/opentracing/specification/blob/master/semantic_conventions.md#span-tags-table